tools/tpm2_nvread.c

```diff
@@ -261,7 +261,7 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
         { "input-session-handle",1,          NULL, 'S' },
```

Comment:
I think is valuable to always include a reference in the message to the commit that introduced the issue, i.e:

Commit 797d90f1e216 ("tpm2_nvread: Add "output" option (-f)") added an
option to define a file path to store the data read from the NV area.

But it didn't add the 'f' option character to the opstring and so isn't
usable. Add it to the optstring.